test/deploy-test.ts

```diff
@@ -160,7 +170,8 @@ const TEST_OPTIONS: DeployOptions = {
   config: TEST_CONFIG,
   message: undefined,
   deployPollInterval: 0,
-  force: "deploy" // default to not re-building and just deploying output as-is
+  force: "deploy", // default to not re-building and just deploying output as-is
+  deployConfigPath: null
```

Comment:
```suggestion
  deployConfigPath: undefined
```

Per earlier comments. We want the default value here, whereas null implies we want no value.